interfaces/builtin/fwupd_test.go

```diff
@@ -155,6 +160,10 @@ func (s *FwupdInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(dbusSpec.SecurityTags(), HasLen, 1)
 
+	// On an all-snaps system, the only UpdateNS rule is applied
```

Comment:
This comment isn't quite right. It doesn't matter if this is an "all-snaps system", it matters if the PermanentSlot is connected to the snap (which you do 9 lines above). Also, this test should be moved up to the apparmorSpec section. I can do this in a follow-up.